doc/devel/MEP/MEP12.rst

```diff
@@ -196,6 +196,6 @@ navigate the gallery. Thus, tags are complementary to this reorganization.
 
 .. [1] http://github.com/matplotlib/matplotlib/pull/714
 .. [2] http://github.com/matplotlib/matplotlib/issues/524
-.. [3] http://matplotlib.1069221.n5.nabble.com/Matplotlib-gallery-td762.html#a33379091
```

Comment:
It automatically redirects you to uppercased variant because this part of url is just a topic of the thread